src/frontend/components/DashboardPaneButtons/index.tsx

```diff
@@ -100,7 +101,18 @@ export const DashboardPaneButtons = ({
       {objectType === modelName.VIDEOS &&
         object.upload_state === uploadState.PENDING &&
         isFeatureEnabled(flags.VIDEO_LIVE) && (
-          <DashboardVideoLiveConfigureButton video={object as Video} />
+          <div>
```

Comment:
```suggestion
          <Fragment>
```
You did not wrap `DashboardVideoLiveConfigureButton` previously, so I think you could use a `<Fragment>` instead of a `<div>`